presto-product-tests/bin/product-tests-suite-5.sh

```diff
@@ -17,10 +17,9 @@ presto-product-tests-launcher/bin/run-launcher test run \
     -- -g storage_formats,hdfs_impersonation,authorization -x iceberg,"${DISTRO_SKIP_GROUP}" -e "${DISTRO_SKIP_TEST}" \
     || suite_exit_code=1
 
-# TODO: enable avro tests once https://github.com/prestosql/presto/issues/3794 is fixed
 presto-product-tests-launcher/bin/run-launcher test run \
     --environment multinode-hive-caching \
-    -- -g hive_caching,storage_formats -x iceberg,avro,"${DISTRO_SKIP_GROUP}" -e "${DISTRO_SKIP_TEST}" \
+    -- -g hive_caching,storage_formats -x iceberg,"${DISTRO_SKIP_GROUP}" -e "${DISTRO_SKIP_TEST}" \
```

Comment:
should https://github.com/prestosql/presto/issues/3794 be closed?

Reply:
will close when tests pass